docs/source/programming_resources/shared/using_android_device/Using-Your-Android-Device.rst

```diff
@@ -4,8 +4,22 @@ Using Your Android Device
 Before you get started with your control system, it is helpful if you
 familiarize yourself with the basic operation of your Android device.
 
+REV Robotics Driver Hub
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Teams who are using the REV Robotics Driver Hub as their DRIVER STATION
+should refer to the `official documentation from REV Robotics <https://docs.revrobotics.com/duo-control/driver-hub-gs>`_ 
+for instructions on how to set up and use the REV Robotics Driver Hub.
+
+Android Smartphone
+~~~~~~~~~~~~~~~~~~
+
+While not recommended, the Competition Manual does allow select Android
+smartphones to be used as a DRIVER STATION or ROBOT CONTROLLER. See rule
```

Comment:
Rephrase to not point to a specific rule number. Rule numbers may shift year to year.